docs/src/api.md

```diff
@@ -30,10 +30,39 @@ allvariables
 
 ```@docs
 observed
+parameter_observed
+ParameterObservedFunction
+```
+
+#### Parameter timeseries
+
+If the index provider contains parameters that change during the course of the simulation
+at discrete intervals, it must implement the following methods to ensure correct
+functioning of [`getu`](@ref) and [`getp`](@ref) for value providers that save the parameter
+timeseries. Note that there can be multiple parameter timeseries, in case different parameters
+change at different times.
```

Comment:
```suggestion
may change at different times.
```

Reply:
Fixed in https://github.com/SciML/SymbolicIndexingInterface.jl/pull/75/commits/9046cd71a4147ac48322914436f422f847c4624f